apply-phphandler-multidomain/switch_php_handler.php

```diff
@@ -0,0 +1,111 @@
+    if ($invalidParams['old-required'] === 'true') echo "[!] Old PHP Handler parameter '--old' must be defined\n";
+    if ($invalidParams['new'] === 'true') echo "[!] New PHP Handler is not installed or enabled: ".$inputOpts['new']."\n";
+    if ($invalidParams['new-required'] === 'true') echo "[!] New PHP Handler parameter '--new' must be defined\n";
+    if ($invalidParams['ngx'] === 'true') echo "[!] Parameter '--ngx' must be boolean: true or false\n";
+    if ($invalidParams['ngxcombination'] === 'true') echo "[!] Parameter '--ngx' can be enabled ONLY with FPM handler: ".$inputOpts['new']."\n";
+    if ($invalidParams['include-disabled'] === 'true') echo "[!] Parameter '--include-disabled' must be boolean: true or false\n";
+    echo "[i] Run utility with '-h' or '--help' for instructions\n";
+    exit("[!] Exiting...\n");
+}
+if (array_key_exists("h", $inputOpts) === true) getHelp();
+if (array_key_exists("help", $inputOpts) === true) getHelp();
+if (array_key_exists("ngx", $inputOpts) === false) $inputOpts['ngx']='false';
+if (array_key_exists("include-disabled", $inputOpts) === false) $inputOpts['include-disabled']='false';
+if (array_key_exists("old", $inputOpts) === false) $invalidParams['old-required'] = 'true';
+if (array_key_exists("new", $inputOpts) === false) $invalidParams['new-required'] = 'true';
+if ($inputOpts['ngx'] !== 'true' && $inputOpts['ngx'] !== 'false') $invalidParams['ngx'] = 'true';
+if ($inputOpts['ngx'] === 'true' && substr($inputOpts['new'], -3) !== 'fpm') $invalidParams['ngxcombination'] = 'true';
+if ($inputOpts['include-disabled'] !== 'true' && $inputOpts['include-disabled'] !== 'false') $invalidParams['include-disabled'] = 'true';
+if (in_array("true", $invalidParams)) handleError($invalidParams);
+
+echo "[i] Information [!] Warning [+] Action [-] Skipped\n";
+echo "----------\n[i] Retrieving installed PHP Handlers...\n";
+$instHandlers = json_decode(shell_exec('plesk bin php_handler --list-json 2>/dev/null'), true);
+$invalidParams['old'] = 'true';
+$invalidParams['new'] = 'true';
+foreach ($instHandlers as $item) {
+	global $invalidParams, $inputOpts;
+	if ($inputOpts['old'] === $item['id'] && $item['status'] === 'enabled') $invalidParams['old'] = 'false';
+	if ($inputOpts['new'] === $item['id'] && $item['status'] === 'enabled') $invalidParams['new'] = 'false';
+}
+if (in_array("true", $invalidParams)) handleError($invalidParams);
+echo "[i] Checking domains assigned to PHP Handler ID ".$handlerMap['cur']."...\n";
+$dbdataCmd = shell_exec("plesk db -Ne \"SELECT d.name, h.php, h.php_handler_id FROM domains d, hosting h WHERE d.id=h.dom_id AND d.htype = 'vrt_hst'\"");
+$dbdata = explode("\n", $dbdataCmd);
+$updatePerformed = 'false';
+$dbdataLength = count($dbdata)-1;
+for ($i = 0; $i < $dbdataLength; $i++) {
+	$tempArray = explode("\t", $dbdata[$i]);
+	$domainName = $tempArray[0];
+	$phpEnabled = $tempArray[1];
+	$phpHandlerId = $tempArray[2];
+	if ($phpHandlerId === $inputOpts['old']) {
```

Comment:
Very hard to read because of the long body of the conditional statement. Switch the condition to the opposite one and use "continue" to reduce the nesting level.